app/src/main/java/org/mozilla/fenix/settings/account/AccountSettingsFragment.kt

```diff
@@ -78,9 +78,37 @@ class AccountSettingsFragment : PreferenceFragmentCompat() {
     override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
         setPreferencesFromResource(R.xml.account_settings_preferences, rootKey)
 
+        accountSettingsStore = StoreProvider.get(this) {
+            AccountSettingsStore(
+                AccountSettingsState(
+                    lastSyncedDate =
+                        if (getLastSynced(requireContext()) == 0L)
+                            LastSyncTime.Never
+                        else
+                            LastSyncTime.Success(getLastSynced(requireContext())),
+                    deviceName = ""
```

Comment:
We actually have a default device name. See `BackgroundServices.kt`, in it there's:
```
private val defaultDeviceName = context.getString(
        R.string.default_device_name,
        context.getString(R.string.app_name),
        Build.MANUFACTURER,
        Build.MODEL
    )
```

Can we turn that into a public helper function `defaultDeviceName(context: Context): String`, in `BackgroundServices`, and use it in both places?